charts/catalog/templates/apiserver-deployment.yaml

```diff
@@ -20,6 +20,7 @@ spec:
         release: "{{ .Release.Name }}"
         heritage: "{{ .Release.Service }}"
     spec:
+      serviceAccountName: server
```

Comment:
The walkthrough test fails because `contrib/rbac-setup.sh` isn't run in `contrib/hack/test_walkthrough.sh`, so the service account additions here break the deployment.

We should either make these `serviceAccountName`s optional, or update both `contrib/hack/test_walkthrough.sh` and the walkthrough docs to require running `contrib/rbac-setup.sh`.